### PR TITLE
[#236] Update services

### PR DESCRIPTION
**Pull request by goreck888**
Add new services
Add new services fields

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -40,10 +45,11 @@
                     access_policies_url: hash["access_policies_url"],
                     places: hash["places"],
                     languages: hash["languages"],
-                    dedicated_for: hash["dedicated_for"],
+                    # dedicated_for: hash["dedicated_for"],
```
Comment (mkasztelnik) [inline db/seeds.rb]: Why this is commented? Should this field be removed?

File: db/schema.rb [hunk 2]
```diff
@@ -143,6 +143,7 @@
     t.text "dedicated_for", null: false
     t.text "terms_of_use_url", null: false
     t.text "access_policies_url", null: false
+    t.text "area", null: false
```
Comment (mkasztelnik) [inline db/schema.rb]: You need to add `area` field into `spec/factories/services.rb` factory to fix specs.

Comment (mkasztelnik) [reply db/schema.rb]: And this is strange - change in `schema.rb` but no migration. How did you lead to such a situation?

**Comment by mkasztelnik** [comment]
A lot of changes not related to this PR is included. Can you fix it?

**Comment by roksanaer** [comment]
and another minor comment. types of users included in the descriptions are not from our list, which goes:

Researchers
VO
Providers
Research project
Business

@agpul please map targeted users to ours so @goreck888 can introduce changes accordingly

I saw that EGI included Research organizations in their eIC description. if other services also have it, we can add it to the list

**Comment by roksanaer** [comment]
>After analysis I am going to change Research project to research organisations. Research project doesnt fit here for me

OK

**Comment by mkasztelnik** [comment]
Also changelog entry is missing

**Comment by roksanaer** [comment]
providers need to have n-n relation model but for tomorrow's purpose it's great. thanks!

**Pull request merged**